chajda/tsquery/augments.py

```diff
@@ -53,114 +58,225 @@ def augments_gensim(lang, word, config=Config(), n=5):
 
     return words
 
-import os
-import fasttext
-import fasttext.util
-from annoy import AnnoyIndex
+
+#################################################################################
+# fasttext
+#################################################################################
 
 # path used for storing/loading fasttext models and annoy indices
 fasttext_annoy_data_path = os.path.dirname(__file__)
 
+# fasttext_models dictionary will hold the loaded fasttext models as values,
+# and the corresponding langauge as keys
```

Comment:
generally describe dictionaries as keys first, then values, since this is the way code will always work;

also the comment should avoid the variable name, since this is redundant (see commend for variable above)